lib/documents.js

```diff
@@ -267,9 +267,7 @@ function probeDocumentsImpl(contentOnly, args) {
   }
 
   var connectionParams = this.client.getConnectionParams();
-  var requestOptions = mlutil.copyProperties(connectionParams);
-  requestOptions.method = 'HEAD';
-  requestOptions.path = mlutil.databaseParam(connectionParams, path, '&');
+  let requestOptions = mlutil.newRequestOptions(connectionParams, path, '&', 'HEAD');
```

Comment:
Sorry, two more small tweaks:

1. It looks like in all these cases, `connectionParams` is only used once. Because fewer variables is almost always better (one exception is when introducing a variable name improves readability, but that's not the case here since it's already clear what `getConnectionParams()` refers to), I would just call `this.client.getConnectionParams()` on line 270 here, and in all the other instances of this code block too. 
2. Unless `requestOptions` is going to be re-assigned to some other value, use `const`. That improves readability by making it clear that `requestOptions` in fact won't be re-assigned to some other value.

Reply:
Done